packages/fabric8-ui/src/app/space/app-launcher/services/app-launcher-mission-runtime.service.spec.ts

```diff
@@ -69,6 +72,14 @@ describe('Service: AppLauncherMissionRuntimeService', () => {
       ],
       providers: [
         AppLauncherMissionRuntimeService,
+        {
+          provide: FeatureTogglesService,
+          useFactory: () => {
+            const mockFeatureTogglesService: jasmine.SpyObj<FeatureTogglesService> = createMock(FeatureTogglesService);
```

Comment:
Is there a test case which actually uses `FeatureToggleService` to check if the feature is enabled or not to show node.js boosters?

Reply:
@rohitkrai03 I added a test case for ```FeatureToggleService```.